### ISSUE TITLE
Training loss don't descend after a sudden drop

### ISSUE THREAD

**Issue by lawsonsli** [callmeray/PointMVSNet#5]
As the picture showing, coarse loss immediately droppped when flow module is added. But after 10 epochs, the loss didn't have a obvious decline.
Is this normal case or I got anything wrong?
![Screenshot from 2019-09-16 22-00-42](https://user-images.githubusercontent.com/18007584/65005559-3e347a80-d8ce-11e9-8372-67a15047e8e9.png)

**Comment by callmeray** [comment]
Yes, that's normal. You should also observe the metrics increase.

**Comment by lawsonsli** [comment]
when calculating coarse loss, there are no scale divided by loss. Do you mean depth_interval?
![Screenshot from 2019-09-16 23-27-29](https://user-images.githubusercontent.com/18007584/65009721-b81f3080-d8db-11e9-900d-938b43d89d69.png)

**Comment by callmeray** [comment]
I mean the metrics which measure the accuracy of the output depth map.
![image](https://user-images.githubusercontent.com/31027932/65009855-f65c2600-d8c2-11e9-9613-03c730414cfd.png)

**Comment by lawsonsli** [comment]
yes, I understand this. flow2, flow1, coarse loss use different metrics in one iteration.
But coarse loss itself uses same metric for all epochs.  It doesn't go down.

**Comment by callmeray** [comment]
Actually it's going down very slowly...

**Comment by lawsonsli** [comment]
Thanks for your great work and nice reply.
